Libraries/LibCore/TCPServer.cpp

```diff
@@ -53,6 +53,7 @@ TCPServer::TCPServer(Object* parent)
 
 TCPServer::~TCPServer()
 {
+    ::close(m_fd);
```

Comment:
Why no `if` here? Because `TCPServer` ctor `assert()`s. However, `LocalServer` does not. We should choose one behavior for consistency.

Reply:
LocalServer doesn't open the socket in the constructor.